### PR TITLE
Add intergation tests for StreamApi::CommitOffsets method

### PR DESCRIPTION
**Pull request by mykolamalik**
Resolves: OLPEDGE-1473

Signed-off-by: Mykola Malik <ext-mykola.malik@here.com>

### CODE REVIEW

File: olp-cpp-sdk-dataservice-read/tests/StreamApiTest.cpp [hunk 1]
```diff
@@ -39,6 +39,22 @@ std::string ApiErrorToString(const ApiError& error) {
   return result_stream.str();
 }
 
+model::StreamOffsets GetStreamOffsets() {
+  static model::StreamOffsets offsets;
```
Comment (andescu) [inline olp-cpp-sdk-dataservice-read/tests/StreamApiTest.cpp]: Why does it need to be static?

Comment (mykolamalik) [reply olp-cpp-sdk-dataservice-read/tests/StreamApiTest.cpp]: To be initialized and filled only once. It will be used for SeekToOffset tests also

Comment (andescu) [reply olp-cpp-sdk-dataservice-read/tests/StreamApiTest.cpp]: This is just complicating the test. You make static something that needs to have a certain lifecycle or has some limitations that force you to use the singleton pattern. In this scenario, I don't see any need to make it static and this only adds additional confusion.
Is there any limitation that forces you to make it static or do you aim for obscurity?

Comment (mykolamalik) [reply olp-cpp-sdk-dataservice-read/tests/StreamApiTest.cpp]: Static variable is removed.

**Pull request merged**